source/_components/homematic.markdown

```diff
@@ -34,28 +34,90 @@ homematic:
 
 Configuration variables (global):
 
-- **interfaces** (*Required*): Configuration for each XML-RPC interface to integrate into Home Assistant.
-- **hosts** (*Optional*): Configuration for each Hub (CCU/Homegear) to integrate into Home Assistant.
-- **local_ip** (*Optional*): IP of device running Home Assistant. Override auto-detected value for exotic network setups.
-- **local_port** (*Optional*): Port for connection with Home Assistant. By default it is randomly assigned.
+{% configuration %}
+interfaces:
+  description: Configuration for each XML-RPC interface to integrate into Home Assistant.
+  required: true
+  type: list
+hosts:
+  description: Configuration for each Hub (CCU/Homegear) to integrate into Home Assistant.
+  required: false
+  type: list
+local_ip:
+  description: IP of device running Home Assistant. Override auto-detected value for exotic network setups.
+  required: false
+  type: string
+local_port:
+  description: Port for connection with Home Assistant. By default it is randomly assigned.
+  required: false
+  type: int
```

Comment:
`integer`

Reply:
Documentation says "int" for the type - is it incorrect? https://developers.home-assistant.io/docs/en/documentation_create_page.html